src/classes/SObjectFieldWrapper.cls

```diff
@@ -0,0 +1,29 @@
+public class SObjectFieldWrapper {
```

Comment:
ordering by parent fields was easy to add, but I also ended up cutting a few of the orderBy methods you had written - hard to imagine needing to order by a field set, you know?  Because we have SO many repositories, and because our implementation of SObjectRepository is still virtual, we actually moved mostly away from FieldSets when initializing repos.  In our factory class, we have a generic method that takes a `List<SObjectField>` and `SObjectType` and uses those to initialize the repo accordingly.  The only exceptions to that are the two repos that I was working on back in March when we originally committed to master (Lead, Opportunity) AND any repo that we have needed to override QueryBuilder methods on - as an example, we created record types for our Contact object (Primary / Emergency) and our PrimaryContactRepo overrides the default implementation for getting the QueryBuilder to add a QueryFilter for primary record type.

Anyway, that was a long aside, all with the purpose of stating that I think that including parent fields in a query's select statement is brilliant and based off of the previous constructors for SObjectRepository, replacing the `List<SObjectField` parameter would mean for easy access in implementing that.  

Finally, with that in mind - I'm curious to see if you have any static solutions for a less unseemly way to load each wrapper with its SObjectField / list of fields!